samples/drivers/counter/alarm/prj.conf

```diff
@@ -1,2 +1,4 @@
 CONFIG_PRINTK=y
 CONFIG_COUNTER=y
+CONFIG_COUNTER_RTC0=y
```

Comment:
Could this be done conditionally in the application Kconfig?

```
config COUNTER_RTC0
	default y if SOC_FAMILY_NRF
```

Something like that.

Reply:
done